---
bool success
float64 score